cg/cli/delete/observations.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import click
+from cg.apps.loqus import LoqusdbAPI
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group("delete")
+def delete_cmd():
+    """Delete variants associated to a case in LoqusDB."""
+    pass
+
+
+@delete_cmd.command()
+@click.option("-c", "--case_id", help="internal case id")
+@click.pass_obj
+def observations(context: CGConfig, case_id: str):
+    """Delete case from LoqusDB and reset loqusdb_id in statusDB."""
+
+    status_db: Store = context.status_db
+    loqusdb_api: LoqusdbAPI = context.loqusdb_api
+
+    # Delete case from loqusdb
+    LOG.info("This will delete all variants in LoqusDB for case: %s", case_id)
+    click.confirm("Do you want to continue?", abort=True)
```

Comment:
It would be nice to also add this:
```
@click.option("-y", "--yes", is_flag=True, help="Skip confirmation")
```
I bet that is repeated across cg. Could you consider making it a global constant, much like the dry-run option. then we do not have to repeat ourselves everywhere (DRY)